Application/Commands/PrivateChats/CreatePrivateChat/CreatePrivateChatRequestHandler.cs

```diff
@@ -10,26 +11,23 @@ public class CreatePrivateChatRequestHandler : RequestHandlerBase,  IRequestHand
     {
         public async Task<PrivateChat> Handle(CreatePrivateChatRequest request, CancellationToken cancellationToken)
         {
-            List<User> users = new();
-            request.UsersId.ForEach(userId => users.Add(Context.Users.Find(userId)
-                                                        ?? throw new EntityNotFoundException(
-                                                            $"User {userId} not found")));
-            User owner = await Context.FindByIdAsync<User>(UserId, cancellationToken);
+            List<UserLookUp> users = new();
+            request.UsersId.ForEach(userId => users.Add(Mapper.Map<UserLookUp>(Context.FindSqlByIdAsync<User>(userId, cancellationToken).Result)));
+            
+            User owner = await Context.FindSqlByIdAsync<User>(UserId, cancellationToken);
             PrivateChat privateChat = new()
             {
                 Title = request.Title,
                 Image = request.Image,
                 Users = users,
-                Messages = new List<Message>(),
                 OwnerId = owner.Id
```

Comment:
Тут можно использовать просто ```UserId ``` 
```suggestion
                OwnerId = UserId
```